frost-ristretto255/src/frost.rs

```diff
@@ -635,18 +909,7 @@ pub fn aggregate(
     signing_shares: &[SignatureShare],
     pubkeys: &PublicKeyPackage,
 ) -> Result<Signature, &'static str> {
-    // TODO(dconnolly): tidy up now that rho is the same for all i's
-
-    let mut bindings: HashMap<u64, Scalar> =
-        HashMap::with_capacity(signing_package.signing_commitments.len());
-
-    let rho = generate_rho(signing_package);
-
-    for comm in signing_package.signing_commitments.iter() {
-        bindings.insert(comm.index, rho);
-    }
-
-    let group_commitment = generate_group_commitment(signing_package, &bindings)?;
+    let group_commitment = GroupCommitment::try_from(signing_package)?;
```

Comment:
Specifically, this line needs to be changed to instead use values sent strictly in round one.

Reply:
That is basically what the `SigningPackage` is, the commitments sent via the Coordinator to all parties in round 1, but I would like to make this even more explicit with `round1::SigningCommitments` and other types, in a following PR

Reply:
Updated in fadb6fe